schema/bom-1.6.proto

```diff
@@ -1150,9 +1152,136 @@ message ModelCard {
+      EnergyMeasureType energyCost = 3;
+      // The CO2 cost or debit equivalent to the total energy cost.
+      optional CO2MeasureType co2CostEquivalent = 4;
+      // The CO2 offset or credit for the CO2 equivalent cost.
+      optional CO2MeasureType co2CostOffset = 5;
+    }
   }
 }
 
+// An activity that is part of a machine learning model development or operational lifecycle.
+enum ActivityType {
+  // a lifecycle activity type whose description does not match currently defined values.
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `other` is our fallback
+  ACTIVITY_TYPE_OTHER = 0;
+  // model design including problem framing, goal definition and algorithm selection.
+  ACTIVITY_TYPE_DESIGN = 1;
+  // model data acquisition including search, selection and transfer.
+  ACTIVITY_TYPE_DATA_COLLECTION = 2;
+  // model data preparation including data cleaning, labeling and conversion.
+  ACTIVITY_TYPE_DATA_PREPARATION = 3;
+  // model building, training and generalized tuning.
+  ACTIVITY_TYPE_TRAINING = 4;
+  // refining a trained model to produce desired outputs for a given problem space.
+  ACTIVITY_TYPE_FINE_TUNING = 5;
+  // model validation including model output evaluation and testing.
+  ACTIVITY_TYPE_VALIDATION = 6;
+  // explicit model deployment to a target hosting infrastructure.
+  ACTIVITY_TYPE_DEPLOYMENT = 7;
+  // generating an output response from a hosted model from a set of inputs.
+  ACTIVITY_TYPE_INFERENCE = 8;
+}
+
+// Describes the physical provider of energy used for model development or operations.
+message EnergyProviderType {
+  // BOM unique reference to the energy provider.
+  optional string bom_ref = 1;
+  // The name of the energy provider.
+  string name = 2;
+  // A description of the energy provider.
+  optional string description = 3;
+  // TThe physical address (location) of the energy provider.BD
+  optional PostalAddressType address = 4;
+  // The energy source for the energy provider.
+  EnergySourceType energySource = 5;
+  // The energy provided by the energy source for an associated activity.
+  EnergyMeasureType energyProvided = 6;
+}
+
+enum EnergySourceType {
+  // energy source: unknown
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `unknown` is our fallback
+  ENERGY_SOURCE_TYPE_UNKNOWN = 0;
+  // energy source: coal
+  ENERGY_SOURCE_TYPE_COAL = 1;
+  // energy source: oil
+  ENERGY_SOURCE_TYPE_OIL = 2;
+  // energy source: natural gas
+  ENERGY_SOURCE_TYPE_NATURAL_GAS = 3;
+  // energy source: propane
+  ENERGY_SOURCE_TYPE_PROPANE = 4;
+  // energy source: nuclear
+  ENERGY_SOURCE_TYPE_NUCLEAR = 5;
+  // energy source: wind
+  ENERGY_SOURCE_TYPE_WIND = 6;
+  // energy source: solar
+  ENERGY_SOURCE_TYPE_SOLAR = 7;
+  // energy source: geothermal
+  ENERGY_SOURCE_TYPE_GEOTHERMAL = 8;
+  // energy source: hydropower
+  ENERGY_SOURCE_TYPE_HYDROPOWER = 9;
+  // energy source: other; not yet defined
+  ENERGY_SOURCE_TYPE_OTHER = 10;
```

Comment:
please move the `ENERGY_SOURCE_TYPE_OTHER` to value `1`

Reply:
It would be desirable for the default to be "other".

Reply:
I do not think so
https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
> It may be tempting to declare this default as a semantically meaningful value but as a general rule, do not, to aid in the evolution of your protocol as new enum values are added over time.

Reply:
done